src/main/resources/org/jenkinsci/plugins/workflow/steps/DeleteDirStep/help.html

```diff
@@ -4,3 +4,9 @@
     To delete a specific directory of a workspace wrap the <code>deleteDir</code>
     step in a <code>dir</code> step.
 </div>
+<br>
+<p>The <a href="https://www.jenkins.io/redirect/pipeline-snippet-generator">Pipeline Snippet Generator</a> generates this example:
```

Comment:
I'm not sure if this is very useful.  I think it would be more helpful if you were building from the previous step to show how the step would look like.  "The following snippet would delete the `./someDir` directory"